### PR TITLE
Allow to overwrite an operator by same name and subject using --force

### PR DESCRIPTION
**Pull request by matthiashanel**
Signed-off-by: Matthias Hanel <mh@synadia.com>

### CODE REVIEW

**Review by aricart — APPROVED** [review]
LGTM

File: cmd/addoperator.go [hunk 1]
```diff
@@ -216,9 +218,15 @@ func (p *AddOperatorParams) PostInteractive(ctx ActionCtx) error {
 func (p *AddOperatorParams) Validate(ctx ActionCtx) error {
 	var err error
 	if p.token != "" {
+		if p.sysAcc {
+			return fmt.Errorf("importing an operator is not compatable with system account generation")
```
Comment (stjepang) [inline cmd/addoperator.go:222]:
```suggestion
			return fmt.Errorf("importing an operator is not compatible with system account generation")
```

Comment (matthiashanel) [reply cmd/addoperator.go]: fixed

Comment (matthiashanel) [reply cmd/addoperator.go]: dang, didn't notice that commit suggestion button

**Pull request merged**